### PR TITLE
Flower Buds now have a countdown for ghosts for when they're going to hatch

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

Flower Buds that make venus human traps

## Why It's Good For The Game

They're two minute timers with no indication, which makes kudzu kinda annoying to spectate.

## Changelog
:cl:
add: Flower Buds now have a countdown for ghosts
/:cl:

### CODE REVIEW

**Comment by IndieanaJones** [comment]
Good improvement on feature, 10/10 speedomerge based

File: code/game/objects/effects/countdown.dm [hunk 1]
```diff
@@ -157,3 +157,13 @@
...
+
+/obj/effect/countdown/flower_bud
+	name = "flower bud countdown"
+
+/obj/effect/countdown/flower_bud/get_value()
+	var/obj/structure/alien/resin/flower_bud_enemy/bud = attached_to
+	if(!istype(bud))
+		return
+	else if(bud.growth_timer)
+		return round(timeleft(bud.timerid), 1)
```
Comment (Timberpoes) [inline code/game/objects/effects/countdown.dm:169]:
```
code\game\objects\effects\countdown.dm:169:error: bud.timerid: undefined var
```
Did you mean to use bud.growth_timer instead?

File: code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm [hunk 2]
```diff
@@ -20,8 +20,12 @@
 	/// The amount of time it takes to create a venus human trap.
 	var/growth_time = 120 SECONDS
 
+	var/growth_timer
+	var/obj/effect/countdown/flower_bud/countdown
```
Comment (Timberpoes) [inline code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm:24]: These need to be documented as per our [documentation guidelines](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)

**Review by Timberpoes — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/countdown.dm (hunk 1 above)
Comment (tralezab) [reply code/game/objects/effects/countdown.dm:169]: yeah, but it definitely doesn't damn work even if the vars are right. hummm...

**Review by Timberpoes — APPROVED** [review]

**Pull request merged**